docs/quickstarts/ratify-on-azure.md

```diff
@@ -3,69 +3,252 @@ title: Ratify on Azure
 sidebar_position: 2
 ---
 
-# Ratify on Azure: Allow only signed images to be deployed on AKS with Notation and Ratify
+# Ratify on Azure: Allow only signed images to be deployed on AKS with Ratify
 
-The signed container images enable users to assure deployments are built from a trusted entity and verify images haven't been tampered with since their creation. The signed image ensures integrity and authenticity before the user pulls an image into any environment and avoid attacks.
+Signing container images ensure their authenticity and integrity. By deploying only signed images on Azure Kubernetes Service (AKS), you can ensure that the images come from a trusted origin and have not been altered since they were created.
 
-Azure Key Vault (AKV) stores a signing key, Notation and the Notation AKV plugin (azure-kv) consumes this key to sign and verify container images and other artifacts. Azure Container Registry (ACR) allows you to store and distribute signed images with signatures.
+With Azure Container Registry (ACR), you can store and distribute images with signatures together. You can use Azure Key Vault (AKV) to keep your signing keys and certificates safe, and then use tools like [Notation](https://github.com/notaryproject/notation) or [Cosign](https://github.com/sigstore/cosign) to sign your container images with them.
 
-This article walks you through an end-to-end workflow of validating and enforcing only signed images are allowed to be deployed on AKS with Notation and Ratify.
+This article walks you through an end-to-end workflow of deploying only signed images on AKS with Ratify.
+
+![e2e workflow diagram](../imgs/e2e-workflow-diagram.png)
```

Comment:
@yizha1  The dimension and resolution of this picture seem changed. For consistency and better visual effect, I would suggest using an SVG file. I added the up-to-date diagram SVG in another PR https://github.com/deislabs/ratify-web/pull/74.